pkg/impersonation/impersonation.go

```diff
@@ -20,18 +21,22 @@ import (
 )
 
 const (
-	impersonationLabel     = "authz.cluster.cattle.io/impersonator"
+	impersonationLabel = "authz.cluster.cattle.io/impersonator"
+	// ImpersonationNamespace is the namespace where impersonation service accounts live.
 	ImpersonationNamespace = "cattle-impersonation-system"
-	ImpersonationPrefix    = "cattle-impersonation-"
+	// ImpersonationPrefix is the prefix for impersonation roles, bindings, and service accounts.
```

Comment:
Phrasing nitpick:
`ImpersonationPrefix is the prefix used for impersonation related resources (roles, bindings, service accounts, etc.)`

Reply:
There is no "etc", there are only these three things. Will leave as is.